pkgs/by-name/re/resonance/package.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, cargo
+, dbus
+, desktop-file-utils
+, fetchFromGitHub
+, glib
+, gst_all_1
+, gtk4
+, libadwaita
+, libxml2
+, meson
+, ninja
+, nix-update-script
+, openssl
+, pkg-config
+, python3
+, python3Packages
+, rustPlatform
+, rustc
+, sqlite
+, stdenv
+, wrapGAppsHook4
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "resonance";
+  version = "0-unstable-2023-06-06";
+
+  src = fetchFromGitHub {
+    owner = "nate-xyz";
+    repo = "resonance";
+    rev = "97826093e22418c0efdb4e61cc75d981bb82c120";
+    hash = "sha256-DgNUjb8+2WTw91OGgFf97YL6lnODtkftYAP/c05RUPI=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    src = finalAttrs.src;
+    hash = "sha256-/v3OokClOk95GOzidBHRkUG7kjHQm35yPeC1n3PzcyM=";
+  };
+
+  nativeBuildInputs = [
+    cargo
+    dbus
+    desktop-file-utils
+    glib
+    gtk4
+    libadwaita
+    libxml2
+    meson
+    ninja
+    openssl
```

Comment:
Some of these seem a bit weird to me. In particular, why would `dbus`, `glib`, `gtk4`, `libadwaita`, `libxml` and `openssl` be needed as `nativeBuildInputs`, rather than `buildInputs`? If you're not sure, I think you should try moving them over and see if it still builds.